app/assets/javascripts/admin/products/services/unit_prices.js.coffee

```diff
@@ -1,6 +1,9 @@
 angular.module("admin.products").factory "UnitPrices", (VariantUnitManager, localizeCurrencyFilter) ->
   class UnitPrices
     @displayableUnitPrice: (price, scale, unit_type, unit_value, variant_unit_name) ->
+      decimal_separator = I18n.toCurrency(.1).substring(1,2)
+      if (decimal_separator == ",")
+        price = price.replace(",", ".")
```

Comment:
IMO this shouldn't be a responsibility of this module and should instead be handled by perhaps `localize_currency.js.coffee` if it's not handled somewhere else already. Duplicating this logic the next time we need to localize a money amount is not an option.

Reply:
Done with `app/assets/javascripts/admin/utils/filters/unlocalize_currency.js.coffee`